docs/global/modules/core/pages/howto/mps-model-server-plugin.adoc

```diff
@@ -0,0 +1,41 @@
+= How-To Run the Light `model-server` in MPS as a Plugin
+:navtitle: mps-model-server-plugin
+
+
+You can run a *light version* of the `model-server` directly in MPS via the dedicated *MPS as Modelix Model Server Plugin*, which is published over at the https://plugins.jetbrains.com/plugin/22834-mps-as-modelix-model-server[JetBrains Marketplace^].
+
+NOTE: More information on the component in general can be found in the xref:core:reference/component-mps-model-server-plugin.adoc[corresponding component reference].
+
+[IMPORTANT]
+====
+A light model-server does not provide the full features as an independent `model-server` instance will.
+More *advanced features* will not work with this light version (e.g., modelix specific repositories/branches, or the xref:howto/metrics.adoc[metrics]).
+
+If you want the full capabilities of the `model-server`, check out xref:howto/usage-model-api-gen-gradle.adoc[how to start it standalone].
+====
+
+
+== How to Install the Plugin
+
+In MPS navigate to "File > Settings > Plugins (left) > Marketplace (top) > Search for "modelix model server" > Install > Restart IDE".
```

Comment:
```suggestion
In MPS navigate to menu:File[Settings > Plugins (left) > Marketplace (top)].
Search for "modelix model server" and press btn:[Install] and afterwards btn:[Restart IDE].
```

That would be what the documentation says for the syntax.

Reply:
The shorthand syntax I used is also correct according to the docs

Reply:
I transitioned it to the menu macro syntax anyway

Reply:
> The shorthand syntax for menu is not on a standards track. You can use it for transient documents, but do not rely on it long term.